src/app/Console/Commands/AddCustomRouteContent.php

```diff
@@ -68,22 +67,7 @@ public function handle()
                 $this->error('Could not write to file: '.$path);
             }
         } else {
-            $command = 'php artisan vendor:publish --provider="Backpack\Base\BaseServiceProvider" --tag=custom_routes';
-
-            $process = new Process($command, null, null, null, 300, null);
-
-            $process->run(function ($type, $buffer) {
-                if (Process::ERR === $type) {
-                    $this->line($buffer);
-                } else {
-                    $this->line($buffer);
-                }
-            });
-
-            // executes after the command finishes
-            if (! $process->isSuccessful()) {
-                throw new ProcessFailedException($process);
-            }
```

Comment:
Should we keep throwing an Exception when it fails?
I don't think so, just asking 😬.

Btw, `Artisan::call()` returns an `int` with the exit code, `0` for success, `1` for error.

Reply:
In tests artisan output is null. They run, but the output is null. That's why I refactored to don't rely on output to get the created files...